local_dynamo.js

```diff
@@ -8,13 +8,20 @@ const EmailDomainBlacklist = require('./src/db/schemas/email_domain_blacklist');
 const IdentityVerificationMethod = require('./src/db/schemas/identity_verification_method');
 const RecoveryMethod = require('./src/db/schemas/recovery_method');
 
-const LOCAL_DYNAMODB_PORT = 7877;
-const TEST_DYNAMODB_PORT = 7879;
+const LOCAL_DYNAMODB_HOST = process.env.LOCAL_DYNAMODB_HOST || 'localhost';
+const LOCAL_DYNAMODB_PORT = process.env.LOCAL_DYNAMODB_PORT || 7877;
+const TEST_DYNAMODB_PORT = process.env.TEST_DYNAMODB_PORT || 7879;
+
+function createTables() {
+    const models = [Account, EmailDomainBlacklist, IdentityVerificationMethod, RecoveryMethod];
```

Comment:
Can we at least add a check here such that if the tables already exist, skip the creation?

Reply:
> Can we at least add a check here such that if the tables already exist, skip the creation?

It fails silently if the tables already exist, it's just a simpler implementation where more complexity is not required. This entire feature is essentially written to 1) run integration tests and 2) mitigate errors in the local dev environment due to no DB running (NB these errors do not affect contract helper usage in a practical way). I've yet to encounter any contributors working on our data layer, which is slated for deprecation in any case.

> And then, in an ideal world, call this from overrideLocalDynamo as well such that the contract helper is smart enough and do not rely on tribal knowledge that tables need to be seeded before it starts

It's not the responsibility of the application to prepare its own database. The expectation is that the infrastructure required by the target environment has been correctly provisioned/initialized. That's why it's handled in the local instances started to satisfy `yarn start` and `yarn test`.

It's also worth pointing out that this initialization is pretty trivial compared to the previous Postgres implementation, which required setting up database users and running ORM migrations (which automatically created missing tables). This would have been performed on a running Postgres instance and not within the application itself.